.github/workflows/dockerized-zeek.yaml

```diff
@@ -45,14 +52,6 @@ jobs:
               - lts
               - '4.0'
               - '4.0.4'
-          - version: 3.2.4-0
-            name: zeek-3
-            lts: false
-            tags:
-              - ${{ github.sha }}
-              - latest
-              - '3'
-              - '3.2'
```

Comment:
Please also adjust the README for this change.

Reply:
Done.